src/seer/automation/codegen/pr_review_coding_component.py

```diff
@@ -45,6 +45,18 @@ def invoke(
             if not final_response:
                 return None
 
+            # Get detailed PR description
+            pr_description = llm_client.generate_structured(
+                messages=agent.memory,
+                prompt=CodingCodeReviewPrompts.format_pr_description_step(
+                    diff_str=request.diff,
+                ),
+                model=GeminiProvider(model_name="gemini-2.0-flash-001"),
+                response_format=CodePrReviewOutput.PrDescription,
+                run_name="Generate PR description",
+                max_tokens=2048,
```

Comment:
The max_tokens=2048 limit for PR description generation might be too restrictive for complex PRs. Consider making this configurable or increasing it to ensure comprehensive descriptions. Also, consider adding error handling for when description generation fails.
```suggestion
max_tokens=4096  # Increased token limit for more detailed descriptions
```

Reply:
this does seem a little bit restrictive, maybe double it to 4096? Makes sense that we want to keep it concise, but we can do that via the prompt, because if it hits the token limit here it'll just error out.